src/main/java/com/google/devtools/build/lib/runtime/commands/CleanCommand.java

```diff
@@ -109,10 +121,10 @@ public ExitCode exec(CommandEnvironment env, OptionsProvider options)
       cleanOptions.cleanStyle = "expunge";
     }
 
-    String cleanBanner = cleanOptions.expunge_async ?
+    String cleanBanner = (cleanOptions.expunge_async || cleanOptions.async) ?
```

Comment:
You also want to duplicate the logic at line 116 to avoid doing async clean on platforms that does not support it.